harvard-cape-peninsula-university-of-technology.csl

```diff
@@ -109,8 +109,8 @@
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group>
```

Comment:
I.e.,

```
      <group delimiter=", ">
        <text macro="author-short"/>
        <text macro="year-date"/>
      </group>
```

You can update the current pull request via the pencil (edit) icon at https://github.com/LaraLibrarian/styles/blob/patch-3/harvard-cape-peninsula-university-of-technology.csl